src/move/emojicoin_dot_fun/sources/hex_codes.move

```diff
@@ -2313,6 +2313,1481 @@ module emojicoin_dot_fun::hex_codes {
+            x"f09fa791f09f8fbfe2808df09f8ea8",                                          // Artist: dark skin tone [1F9D1 1F3FF 200D 1F3A8]
+            x"f09fa791f09f8fbbe2808df09f8ea8",                                          // Artist: light skin tone [1F9D1 1F3FB 200D 1F3A8]
+            x"f09fa791f09f8fbde2808df09f8ea8",                                          // Artist: medium skin tone [1F9D1 1F3FD 200D 1F3A8]
+            x"f09fa791f09f8fbee2808df09f8ea8",                                          // Artist: medium-dark skin tone [1F9D1 1F3FE 200D 1F3A8]
+            x"f09fa791f09f8fbce2808df09f8ea8",                                          // Artist: medium-light skin tone [1F9D1 1F3FC 200D 1F3A8]
+            x"f09fa791e2808df09f9a80",                                                  // Astronaut [1F9D1 200D 1F680]
+            x"f09fa791f09f8fbfe2808df09f9a80",                                          // Astronaut: dark skin tone [1F9D1 1F3FF 200D 1F680]
+            x"f09fa791f09f8fbbe2808df09f9a80",                                          // Astronaut: light skin tone [1F9D1 1F3FB 200D 1F680]
+            x"f09fa791f09f8fbde2808df09f9a80",                                          // Astronaut: medium skin tone [1F9D1 1F3FD 200D 1F680]
+            x"f09fa791f09f8fbee2808df09f9a80",                                          // Astronaut: medium-dark skin tone [1F9D1 1F3FE 200D 1F680]
+            x"f09fa791f09f8fbce2808df09f9a80",                                          // Astronaut: medium-light skin tone [1F9D1 1F3FC 200D 1F680]
+            x"e29b93efb88fe2808df09f92a5",                                              // Broken chain [26D3 FE0F 200D 1F4A5]
+            x"f09f8d84e2808df09f9fab",                                                  // Brown mushroom [1F344 200D 1F7EB]
+            x"f09fa791e2808df09f8db3",                                                  // Cook [1F9D1 200D 1F373]
+            x"f09fa791f09f8fbfe2808df09f8db3",                                          // Cook: dark skin tone [1F9D1 1F3FF 200D 1F373]
+            x"f09fa791f09f8fbbe2808df09f8db3",                                          // Cook: light skin tone [1F9D1 1F3FB 200D 1F373]
+            x"f09fa791f09f8fbde2808df09f8db3",                                          // Cook: medium skin tone [1F9D1 1F3FD 200D 1F373]
+            x"f09fa791f09f8fbee2808df09f8db3",                                          // Cook: medium-dark skin tone [1F9D1 1F3FE 200D 1F373]
+            x"f09fa791f09f8fbce2808df09f8db3",                                          // Cook: medium-light skin tone [1F9D1 1F3FC 200D 1F373]
+            x"f09f91a8e2808de29da4efb88fe2808df09f91a8",                                // Couple with heart: man, man [1F468 200D 2764 FE0F 200D 1F468]
+            // x"f09f91a8f09f8fbfe2808de29da4efb88fe2808df09f91a8f09f8fbf",                // Couple with heart: man, man, dark skin tone [1F468 1F3FF 200D 2764 FE0F 200D 1F468 1F3FF]
```

Comment:
Are we able to uncomment these if we go with this approach?

https://github.com/econia-labs/emojicoin-dot-fun/pull/35#discussion_r1587728380

Reply:
Noting here that these are disabled due to package size restrictions, not execution limits

The Python script should be updated to generate this and eliminate permutations in the case of certain long bases

Reply:
Done!